py/server/deephaven/pandas.py

```diff
@@ -201,12 +201,15 @@ def _map_na(array: [np.ndarray, pd.api.extensions.ExtensionArray]):
     return array
 
 
-def to_table(df: pd.DataFrame, cols: List[str] = None) -> Table:
+def to_table(df: pd.DataFrame, cols: List[str] = None, infer_objects: bool = True) -> Table:
     """Creates a new table from a pandas DataFrame.
 
     Args:
         df (DataFrame): the pandas DataFrame instance
         cols (List[str]): the dataframe column names, default is None which means including all columns in the DataFrame
+        infer_objects (bool): whether to convert columns of object types in the DataFrame to the best possible types
+            supporting pd.NA first before creating the table, default is True. When True, pandas convert_dtypes() method
+            is called to perform the conversion. Note that any conversion will make a copy of the data.
```

Comment:
```suggestion
        infer_objects (bool): whether to infer the best possible column types for columns in the DataFrame before creating the table, default is True. When True, pandas convert_dtypes() method
            is called to perform the conversion. Note that any conversion will make a copy of the data.
```

I made this suggestion because the pandas docs wording feels kind of wonky, so I suggested something more clear.